### PR TITLE
Move workflow models to their own json files [OC-87]

### PR DESCRIPTION
**Pull request by birdbrained**

### CODE REVIEW

File: client/app/serializers/workflow.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import DS from 'ember-data';
+
+
+export default DS.JSONSerializer.extend({
+
+    isSuccess: function(status, headers, payload) {
+        return status >= 200 && status < 300 || status === 304;
```
Comment (samanehsan) [inline client/app/serializers/workflow.js]: Can you clarify what this is doing and why it is necessary?

**Comment by samanehsan** [comment]
@birdbrained just to clarify, this is a temporary refactor until the workflows are in a postgres model right?

**Comment by birdbrained** [comment]
It depends on what happens with workflows. This is better than having it in the route, but it may be better to have workflows provide the JSON for this. It may be better to keep it separate for now, as there's no pressing need for it to be in postgres because were not querying any of this.

**Comment by samanehsan** [comment]
Ok, is there any advantage to loading the json from the ember store instead of importing the JSON directly into the route?

File: client/app/serializers/workflow.js (hunk 1 above)
Comment (birdbrained) [inline client/app/serializers/workflow.js]: If it's cached, it'll return 304 Not Modified, this lets ember know that its still successful.

**Comment by birdbrained** [comment]
Only thing really is that it means we don't load every JSON config into the browser if only we need one.

**Pull request merged**